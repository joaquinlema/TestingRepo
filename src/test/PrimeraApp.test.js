import React from 'react';
import { render } from '@testing-library/react';
import PrimeraApp from '../PrimeraApp';

describe('Pruebas primera app react', () => {

    test('debe mostrar el titulo y subtitulo', () => {

        const saludo = 'hola joaco';

        const {getByText} = render(<PrimeraApp saludo='hola joaco'/>);

        expect(getByText(saludo)).toBeInTheDocument();
    });

});
