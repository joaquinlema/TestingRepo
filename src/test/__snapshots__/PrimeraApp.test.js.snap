// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas primera app react debe mostrar titulo enzime  1`] = `
<Fragment>
  <h1>
     
    hola joaco123
     
  </h1>
  <p>
    Soy un subtitulo
  </p>
</Fragment>
`;
